MANUAL.md

```diff
@@ -96,11 +96,25 @@ You can install and run this wizard like this:
     `rmt-cli products disable SLES/15/x86_64 1743`
 
   * `rmt-cli products show <id | string>`:
-  	Displays product with all its repositories and their attributes.
+    Displays product with all its repositories and their attributes.
 
-  	Examples:
+    Examples:
+
+    `rmt-cli products show SLES/15/x86_64`
+
+  * `rmt-cli clean packages`:
+    Removes locally mirrored stale files and their database entries.
+    A file is considered *stale* if it matches all the following characteristics:
+
+      * It exists in a repository directory with primary and deltainfo metadata files;
```

Comment:
Maybe mention here which metadata file is meant by this, I guess it's `repomd.xml` because that's also what printed when running `rmt-cli clean packages`.